app/views/groups/index.html.erb

```diff
@@ -20,22 +20,22 @@
               dark:divide-gray-700
             "
           >
-          <% @groups.each do |group| %>
-            <tr
-              class="
+            <% @groups.each do |group| %>
+              <tr
+                class="
                   text-sm
                   font-normal
                   text-gray-500
                   hover:bg-gray-50
                   dark:hover:bg-gray-700
```

Comment:
Can you get rid of these hovers here so that it matches how you wanted the projects table

Reply:
Addressed in 313ea06

Reply:
Thanks :)